### ISSUE TITLE
glycan of the {day, week, month}

### ISSUE THREAD

**Issue by kiyoko** [glytoucan/glytoucan.github.io#62]
Might be nice to have a "Glycan of the month" like PDB:
[http://www.rcsb.org/pdb/home/home.do](http://www.rcsb.org/pdb/home/home.do)

**Comment by glycosciences** [comment]
I've thought about something like that as well once in a while, but if you want to do this in the way the PDB does, it means quite some work, because someone has to write up some lines of text why this glycan is of interest. This will be quite easy in the beginning, because there are some glycans that can be easily described (such as the N-glycan core, the Lewis motifs, etc.), but these well-known ones are rather limited, and for less well-known ones it takes more time to find relevant information and write up a summary.

**Comment by kiyoko** [comment]
That's true, if we wanted to do it in so much detail.  However, since we have such few metadata, we may be able to do it with little work.  For example, select the glycans that have the most connections to other database, or the substructures (motifs) that are most used, and then iteratively select those that are second in line, third in line, etc.  For explanations we can display the information that led to its selection, and show its Related Data to other databases or motifs, etc.
